hack/generated/pkg/reconcilers/azure_deployment_reconciler.go

```diff
@@ -593,90 +606,39 @@ func (r *AzureDeploymentReconciler) MonitorDeployment(ctx context.Context) (ctrl
 		return ctrl.Result{}, errors.Wrapf(err, "getting deployment %q from ARM", deploymentID)
 	}
 
-	var status genruntime.FromARMConverter
-	if deployment.IsSuccessful() {
-		// TODO: There's some overlap here with what Update does
-		if len(deployment.Properties.OutputResources) == 0 {
-			return ctrl.Result{}, errors.Errorf("template deployment didn't have any output resources")
-		}
-
-		resourceID, idErr := deployment.ResourceID()
-		if idErr != nil {
-			return ctrl.Result{}, errors.Wrap(idErr, "getting resource ID from resource")
-		}
-
-		s, _, statusErr := r.getStatus(ctx, resourceID)
-		if statusErr != nil {
-			return ctrl.Result{}, errors.Wrap(statusErr, "getting status from ARM")
-		}
+	r.log.V(4).Info("Monitoring deployment", "action", string(CreateOrUpdateActionMonitorDeployment), "id", deploymentID, "state", deployment.ProvisioningStateOrUnknown())
+	r.recorder.Event(r.obj, v1.EventTypeNormal, string(CreateOrUpdateActionMonitorDeployment), fmt.Sprintf("Monitoring Azure deployment ID=%q, state=%q", deploymentID, deployment.ProvisioningStateOrUnknown()))
```

Comment:
Suggest formatting the parameters across separate lines
```suggestion
	r.recorder.Event(
		r.obj, 
		v1.EventTypeNormal, 
		string(CreateOrUpdateActionMonitorDeployment), 
		fmt.Sprintf("Monitoring Azure deployment ID=%q, state=%q", deploymentID, deployment.ProvisioningStateOrUnknown()))
```

Reply:
Did this - also with the above line which was basically just as long